### ISSUE TITLE
Create Kubernetes specific validation functions to use all over

### ISSUE THREAD

**Issue by helgi** [deis/controller#1070]
Create validation functions for the various Kubernetes requirements

Such as validation what is used to validate the various length and char requirements. Basically DNS validation.

Labels are 63 chars, adhere to DNS specs, etc etc

This would allow a few models to have more unified validation functions, especially the tags validation could do with it.

Question is if this lives inside of the `scheduler` or not? Problem with it living there is at the end it needs to raise `ValidationError` for the serializers and friends, but worth investigating

**Comment by bacongobbler** [comment]
perhaps this is a good segway to create a `kubernetes-py` client lib that encapsulates all this behaviour for us.

**Comment by helgi** [comment]
Yeah, initial thought was to add those to the `util` part of `scheduler` to have a `kubernetes-py` like functionality - Probably is enough.

**Comment by Cryptophobia** [comment]
This issue was moved to teamhephy/controller#56
